nixos/modules/services/web-apps/matomo.nix

```diff
@@ -164,6 +164,7 @@ in {
             # Copy config folder
             chmod g+s "${dataDir}"
             cp -r "${cfg.package}/share/config" "${dataDir}/"
+            mkdir "${dataDir}/misc"
```

Comment:
```suggestion
            mkdir -p "${dataDir}/misc"
```
I'd suggest using `-p` parameter as the `matomo-setup-update.service` now fails if the folder already exists.

Reply:
Done, thank you!